pyscriptjs/tests/integration/test_03_element.py

```diff
@@ -1,9 +1,12 @@
-from .support import PyScriptTest, skip_worker
+import pytest
+
+from .support import PyScriptTest
 
 
 class TestElement(PyScriptTest):
     """Test the Element api"""
 
+    @pytest.mark.skip("FIXME: Element missing from PyScript")
```

Comment:
If possible I'd like to add a deprecation cycle before we remove it. If we get #1650 in (or a comparable PR where we can have all the python packages in) we can add pydom and, eventually, `Element` and a few other [deprecated] APIs that we can remove in the following release.

Definitely another topic for us to chat later and decide

Reply:
I think the goal was to break and the community agreed that's what we should do. If we bring back everything we don't want anymore this is not PyScript next. See my previous comment about deprecating in PyScript classic, not in next.

This was agreed and I believe commonly understood until this comment. We don't have `Element` and we are willing to break everything we want to break.

@tedpatrick please confirm what I am saying is what we've discussed, thanks!

Reply:
To clarify, if we bring back all the legacy *to deprecate* we won't have a release that's small and fast and we introduce a lot of code that doesn't belong to next so I hope we'll take the suggested approach or we write a migration pattern, which is partially what I also did in the documentation.

Binging in *Element* was never on the plate and in your release list there's no mention to it neither ... here you are just underlying what I've mentioned about legacy tests: it will take more time to rewrite them all than using the *polyscript* approach which works great already and it's based on the new stack/architecture already.

That being said, if the goal here is to understand what works and what doesn't using messages that tell another story ("display is broken ...") won't help anyone ... the tests are broken because target a different project, somehow, or better, a legacy that we decided to change. I hope we can fix tests instead, but it surely takes time.